features/ship/always_merge/current_branch/undo_warning.feature

```diff
@@ -0,0 +1,22 @@
+Feature: git-town undo prints a warning message for a merge commit
+
+  Background:
+    Given a Git repo with origin
+    And the branches
+      | NAME    | TYPE    | PARENT | LOCATIONS     |
+      | feature | feature | main   | local, origin |
+    And the current branch is "feature"
+    And the commits
+      | BRANCH  | LOCATION      | MESSAGE        |
+      | feature | local, origin | feature commit |
+    And Git setting "git-town.ship-strategy" is "always-merge"
+    When I run "git-town ship" and close the editor
+    Then the current branch is now "main"
```

Comment:
What is the purpose of this step in the context of this test, which verifies the message printed to the user?

Reply:
Are you asking about the verification that we are on the main branch? This is a very simple way to make sure the command actually executed. This verifies that the test makes sense.

If the command hasn't executed, then the developer may incorrectly focus on debugging nonsense output from `undo` instead of focusing on why `git-town ship` didn't work. I added a comment for clarification, PTAL.

Reply:
Yes, I was wondering what the step to verify the current branch does here.

This gives me an idea: the `I ran` step should always fail if the exit code is non-zero. I'll implement this in a separate PR.